doc/lua_api.txt

```diff
@@ -4918,6 +4918,15 @@ Misc.
       of the creative mode setting, checks for "sneak" to set the `invert_wall`
       parameter and `prevent_after_place` set to `true`.
 
+* `minetest.calculate_knockback(player, hitter, time_from_last_punch,
```

Comment:
calculate implies that it doesn't modify state. I suggest calling it `apply_knockback`

Reply:
it indeed does not modify state